jaeger_client/constants.py

```diff
@@ -33,10 +33,10 @@
 DEFAULT_FLUSH_INTERVAL = 1
 
 # Name of the HTTP header used to encode trace ID
-TRACE_ID_HEADER = b'uber-trace-id'
```

Comment:
So here's the scenario in Python 2.7, which I believe reflects what was happening:

```
>>> x=b'x'
>>> x
'x'
>>> y=u'y'
>>> y
u'y'
>>> b=bytes(chr(255))
>>> b
'\xff'
>>> x+b
'x\xff'
>>> y+b
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xff in position 0: ordinal not in range(128)
```

The code in httplib.py looks like this
```python
def _send_output(self, message_body=None):
        . . .
        if isinstance(message_body, str):
            msg += message_body
```

Now, I don't know why `message_body` contained byte sequence that wasn't a valid Unicode, but it did happen in production code (as I mention, in the image upload, fwiw). It's my understanding that normally that HTTP request's buffer was composed of non-Unicode string, which shown in the example to be OK with concatenating non-Unicode sequence (`x+b`). However, when tracing headers were added and those headers were defined without `b` prefix, they we automatically converted into `u` string. When they were appended to the buffer, it turns the buffer into `u`, e.g.
```
>>> x+y
u'xy'
```
And then later when the body with invalid seq is appended, Python tries to parse it as Unicode and blows up.

For this reason I had to declare the headers as `b`, non-Unicode strings. It may not be an issue in Python3, but will definitely be an issue in Python 2.7. So perhaps these assignments can be made conditional using `six.PY3`